nirc_ehr/src/org/labkey/nirc_ehr/columnTransform/NIRCUserCreateTransform.java

```diff
@@ -52,8 +52,9 @@ private User createUser(String name)
         UserSchema us = QueryService.get().getUserSchema(getContainerUser().getUser(), getContainerUser().getContainer(), NIRC_EHRUserSchema.SCHEMA_NAME);
 
         SQLFragment sql = new SQLFragment("SELECT email, displayName, officePhone, homePhone, cellPhone, positionName FROM ").append(Objects.requireNonNull(us.getTable("Staff")), "st");
-        sql.append(" WHERE (lower(firstName) = '").append(nameParts[0].replace("'", "''")).append("' AND lower(lastName) = '")
-                .append(nameParts[1].replace("'", "''")).append("') LIMIT 1");
+        sql.append(" WHERE (lower(firstName) = ? AND lower(lastName) = ?) LIMIT 1");
```

Comment:
@labkey-jeckels They have some users with apostrophes in their names. Is this handled when adding them as parameters?

Reply:
Exactly. That's one of the big advantages of using JDBC parameters - you don't need to worry about escaping the content.